test/coding/test_default_policy_override.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+from conftest import skip_docker
+
+from autogen.code_utils import is_docker_running
+from autogen.coding.docker_commandline_code_executor import DockerCommandLineCodeExecutor
+
+
+@pytest.mark.skipif(
+    skip_docker or not is_docker_running(), reason="Docker is not running or requested to skip Docker tests"
```

Comment:
There's currently a bug in the CI where the docker test fails silently, the fix is in this PR https://github.com/microsoft/autogen/pull/2490, so I would recommend waiting for the fix to get merged before. The fix is essentially 
```suggestion
    skip_docker or not is_docker_running() or not decide_use_docker(use_docker=None), reason="Docker is not running or requested to skip Docker tests"
```

Reply:
The fix is merged, you can do once the test is moved to `test/coding/test_commandline_code_executor.py`:
```suggestion
    skip_docker_test, reason="Docker is not running or requested to skip Docker tests"
```